src/transformers/models/bart/modeling_flax_bart.py

```diff
@@ -218,15 +217,15 @@
 """
 
 
-def shift_tokens_right(input_ids: np.array, pad_token_id: int, decoder_start_token_id: int) -> np.ndarray:
+def shift_tokens_right(input_ids: jnp.array, pad_token_id: int, decoder_start_token_id: int) -> jnp.ndarray:
     """
     Shift input ids one token to the right.
     """
-    shifted_input_ids = np.zeros_like(input_ids)
+    shifted_input_ids = jnp.zeros_like(input_ids)
     shifted_input_ids[:, 1:] = input_ids[:, :-1]
```

Comment:
We can't slice jnp arrays 😉 You'll need to do `.at` followed by `.set` as you did for mbart - as suggested, it might be easier to open these changes in a new PR to track the new code independently of this PR (see https://github.com/huggingface/transformers/pull/23356#pullrequestreview-1426905747)

Reply:
Oh, sorry

Reply:
I fixed it by committing directly!🙏